pkg/shell/indexes.js

```diff
@@ -460,7 +460,7 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
                 title = _("Connecting to the machine");
                 message = "";
             } else {
-                title = _("Couldn't connect to the machine");
+                title = _("Not connected to the machine");
```

Comment:
Since we're poking at this section anyway... :)
Should we call this **host**, as this is what they are called in the host switcher sidebar?

Reply:
Done.